src/main/java/com/fasterxml/jackson/datatype/jsr310/ser/ZonedDateTimeSerializer.java

```diff
@@ -8,6 +9,8 @@
 
     protected ZonedDateTimeSerializer() {
         super(ZonedDateTime.class, dt -> dt.toInstant().toEpochMilli(),
-                ZonedDateTime::toEpochSecond, ZonedDateTime::getNano);
+                ZonedDateTime::toEpochSecond, ZonedDateTime::getNano,
+                // ISO_ZONED_DATE_TIME is not the ISO format, it is an extension of it
+                dt -> DateTimeFormatter.ISO_OFFSET_DATE_TIME.format(dt));
```

Comment:
Shouldn't the lambda here check for a setting to determine whether to use `ISO_OFFSET_DATE_TIME` or `ISO_ZONED_DATE_TIME`? Or does that have to come at a later date after changes are made to the Jackson core?